modules/deleting-the-cluster-logging-operator-by-using-the-cli.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * uninstalling/uninstalling-logging.adoc
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-07-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-the-cluster-logging-operator-by-using-the-cli_{context}"]
+= Deleting the {clo} by using the CLI
+
+Delete the {clo} by using the {oc-first}.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+
+. Delete the `ClusterLogForwarder` custom resource (CR) by running the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc delete ClusterLogForwarder instance -n openshift-logging
```

Comment:
oc delete ClusterLogForwarder <name> -n openshift-logging

Reply:
The serviceaccount need to be deleted too.
oc adm policy remove-cluster-role-from-user lokistack-tenant-logs -z <serviceAccountName>
oc adm policy remove-cluster-role-from-user collect-application-logs -z <serviceAccountName>
oc adm policy remove-cluster-role-from-user collect-infrastructure-logs -z <serviceAccountName>
oc adm policy remove-cluster-role-from-user collect-audit-logs -z <serviceAccountName>
oc adm policy remove-cluster-role-from-user logging-collector-logs-writer  -z <serviceAccountName>
oc delete serviceaccount <serviceAccountName>

Reply:
delete LogFileMetricExporter  if it is deloyed.
oc -n openshift-logging delete LogFileMetricExporter --all